packages/mobile/index.js

```diff
@@ -6,6 +6,10 @@ import App from 'src/app/App'
 import { installSentry } from 'src/sentry/Sentry'
 import { Sentry } from 'react-native-sentry'
 import { onBackgroundNotification } from 'src/firebase/firebase'
+import { YellowBox } from 'react-native'
+
+// To avoid "Remote debugger in background tab" warning
+YellowBox.ignoreWarnings(['Remote debugger'])
```

Comment:
Can you add this to the existing list here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/app/App.tsx#L24

Reply:
done